### PR TITLE
Functional tests - Add new test 'View guest from orders page'

### PR DESCRIPTION
**Pull request by boubkerbribri**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.7.x
| Description?  | Add new test 'View guest from orders page'
| Type?         | refacto
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | no
| How to test?  | `URL_FO=shopUrl/ TEST_PATH='functional/BO/02_orders/01_orders/09_viewGuest`

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/20318)

### CODE REVIEW

**Review by SimonGrn — DISMISSED** [review]

**Comment by NeOMakinG** [comment]
Travis looks red @boubkerbribri

**Comment by boubkerbribri** [comment]
> Travis looks red @boubkerbribri

Should be good now, I deleted a repeated selector, i thought it was not used.

**Review by NeOMakinG — DISMISSED** [review]

**Review by SimonGrn — DISMISSED** [review]

File: tests/UI/campaigns/sanity/06_checkoutFO/01_orderProduct.js [hunk 1]
```diff
@@ -95,7 +95,7 @@ describe('Order a product and check order confirmation', async () => {
     const isCheckoutPage = await checkoutPage.isCheckoutPage(page);
     await expect(isCheckoutPage, 'Browser is not in checkout Page').to.be.true;
 
-    const isStepPIComplete = await checkoutPage.isStepCompleted(page, checkoutPage.personalInformationStepSection);
+    const isStepPIComplete = await checkoutPage.isStepCompleted(page, checkoutPage.personalInformationStepForm);
     await expect(isStepPIComplete, 'Step Personal information is not complete').to.be.true;
```
Comment (Progi1984) [inline tests/UI/campaigns/sanity/06_checkoutFO/01_orderProduct.js]:
```suggestion
    const isStepPersonalInformationComplete = await checkoutPage.isStepCompleted(page, checkoutPage.personalInformationStepForm);
    await expect(isStepPersonalInformationComplete, 'Step Personal information is not complete').to.be.true;
```

**Review by Progi1984 — COMMENTED** [review]
A small comment

**Review by Progi1984 — APPROVED** [review]

**Review by SimonGrn — APPROVED** [review]

**Comment by matks** [comment]
Thank you @boubkerbribri

**Pull request merged**